ConfigSpace/hyperparameters.pyx

```diff
@@ -1663,8 +1668,7 @@ cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
     cdef public nfhp
     cdef normalization_constant
 
-
-    def __init__(self, name: str, mu: int, sigma: Union[int, float],
+    def __init__(self, name: str, mu: Union[int, float], sigma: Union[int, float],
```

Comment:
Seems you explicitly type allow `float`? How come?

Reply:
My thinking is that even if we're drawing integers from a normal distribution, there is no requirement that the mean of the distribution be an integer. For instance, a user might want 4 and 5 to be chosen with equal probability and so place the mean at 4.5.